test/firrtl/lower-to-rtl.mlir

```diff
@@ -37,19 +37,23 @@
     // CHECK: %9 = rtl.concat %5, %8 : (i4, i4) -> i8
     %6 = firrtl.cat %4, %5 : (!firrtl.uint<4>, !firrtl.uint<4>) -> !firrtl.uint<8>
 
-    // CHECK-NEXT: firrtl.connect %out1, %8 : !firrtl.flip<uint<4>>, i4
+    // CHECK-NEXT: rtl.connect %2, %5 : i4
+    firrtl.connect %2, %4 : !firrtl.uint<4>, !firrtl.uint<4>
+
+    // CHECK-NEXT: %10 = firrtl.stdFlippedIntCast %out1 : (!firrtl.flip<uint<4>>) -> i4
+    // CHECK-NEXT: rtl.connect %10, %8 : i4
     firrtl.connect %out1, %5 : !firrtl.flip<uint<4>>, !firrtl.uint<4>
 
-    // CHECK-NEXT: %10 = rtl.wire {name = "test-name"} : i4
+    // CHECK-NEXT: %11 = rtl.wire {name = "test-name"} : i4
     firrtl.wire {name = "test-name"} : !firrtl.uint<4>
 
-    // CHECK-NEXT: %11 = rtl.wire : i2
+    // CHECK-NEXT: %12 = rtl.wire : i2
     firrtl.wire : !firrtl.uint<2>
 
-    // CHECK-NEXT: %12 = rtl.wire : !firrtl.vector<uint<1>, 13>
+    // CHECK-NEXT: %13 = rtl.wire : !firrtl.vector<uint<1>, 13>
     %_t_2 = firrtl.wire : !firrtl.vector<uint<1>, 13>
 
-    // CHECK-NEXT: %13 = rtl.wire : !firrtl.vector<uint<2>, 13>
+    // CHECK-NEXT: %14 = rtl.wire : !firrtl.vector<uint<2>, 13>
```

Comment:
should change these tests so that they use the pattern matching in FileCheck:
https://llvm.org/docs/CommandGuide/FileCheck.html#filecheck-regex-matching-syntax

Reply:
Made this an issue. Will make a new branch/PR for fixing test cases.